src/order/conditionally_complete_lattice.lean

```diff
@@ -132,35 +132,32 @@ instance complete_linear_order.to_conditionally_complete_linear_order_bot {α :
 section
 open_locale classical
 
-/-- A well founded linear order with a bottom element is conditionally complete. -/
-@[reducible] noncomputable def well_founded.conditionally_complete_linear_order_with_bot
-  {α : Type*} [i₁ : linear_order α] [i₂ : order_bot α] (h : well_founded ((<) : α → α → Prop)) :
+/-- A well founded linear order is conditionally complete, with a bottom element. -/
+@[reducible] noncomputable def well_founded.conditionally_complete_linear_order_bot
```

Comment:
Note that the current name is what it is due to dot notation; but this change makes that irrelevant.

Reply:
What would a better name be? Maybe `is_well_order.conditionally_complete_linear_order_bot`?

Reply:
Yes, `is_well_order` sounds good.
Thanks!
bors d+